extensions/kill-process/package.json

```diff
@@ -101,6 +101,14 @@
       "default": true,
       "type": "checkbox"
     },
+    {
+      "description": "Back to root search after killing a process",
+      "label": "Back to Root Search",
```

Comment:
I would rename these two as "Go back to..."

Reply:
does "Pop to Root Search" make more sense to make consistent with the wording in the Advanced Raycast Setting.